admin-dev/themes/new-theme/js/pages/order/create/addresses-renderer.js

```diff
@@ -36,68 +36,74 @@ export default class AddressesRenderer {
    * @param {Array} addresses
    */
   render(addresses) {
-    let deliveryAddressDetailsContent = '';
-    let invoiceAddressDetailsContent = '';
-
-    const $deliveryAddressDetails = $(createOrderPageMap.deliveryAddressDetails);
-    const $invoiceAddressDetails = $(createOrderPageMap.invoiceAddressDetails);
-    const $deliveryAddressSelect = $(createOrderPageMap.deliveryAddressSelect);
-    const $invoiceAddressSelect = $(createOrderPageMap.invoiceAddressSelect);
-
-    const $addressesContent = $(createOrderPageMap.addressesContent);
-    const $addressesWarningContent = $(createOrderPageMap.addressesWarning);
-
-    $deliveryAddressDetails.empty();
-    $invoiceAddressDetails.empty();
-    $deliveryAddressSelect.empty();
-    $invoiceAddressSelect.empty();
-
+    this._cleanAddresses();
     if (addresses.length === 0) {
-      $addressesWarningContent.removeClass('d-none');
-      $addressesContent.addClass('d-none');
+      this._hideAddressesContent();
+      this._showEmptyAddressesWarning();
+      this._showAddressesBlock();
 
       return;
     }
 
-    $addressesContent.removeClass('d-none');
-    $addressesWarningContent.addClass('d-none');
+    this._showAddressesContent();
+    this._hideEmptyAddressesWarning();
 
-    for (const key in Object.keys(addresses)) {
+    for (const key in addresses) {
       const address = addresses[key];
 
-      const deliveryAddressOption = {
-        value: address.addressId,
-        text: address.alias,
-      };
-
-      const invoiceAddressOption = {
-        value: address.addressId,
-        text: address.alias,
-      };
-
-      if (address.delivery) {
-        deliveryAddressDetailsContent = address.formattedAddress;
-        deliveryAddressOption.selected = 'selected';
-      }
+      this._renderDeliveryAddress(address);
+      this._renderInvoiceAddress(address);
+    }
 
-      if (address.invoice) {
-        invoiceAddressDetailsContent = address.formattedAddress;
-        invoiceAddressOption.selected = 'selected';
-      }
+    // @todo: router.generate(admin_addresses_create) when pr #15300 merged
+    $(createOrderMap.addressAddBtn).prop('href', '#');
+    this._showAddressesBlock();
+  }
 
-      $deliveryAddressSelect.append($('<option>', deliveryAddressOption));
-      $invoiceAddressSelect.append($('<option>', invoiceAddressOption));
+  /**
+   * Renders delivery address content
+   *
+   * @param address
+   *
+   * @private
+   */
+  _renderDeliveryAddress(address) {
```

Comment:
Could you stop using useless `_` and `@private` annotation please, it will be remove with the eslint-airbnb-base
ping @NeOMakinG

Reply:
https://eslint.org/docs/2.0.0/rules/no-underscore-dangle

This is I think integrated into airbnb rules, we should not use underscores in variables names, function..

Reply:
This seems it can be allowed with an option Although I agree we could get rid of the `_` however I don't see why we should remove the `private` annotation Even though it's just for information it's convenient to understand the scope of the functions and the way they should be used

Reply:
Thats said considering the number of function starting with `_` it shouldn't be blocking this PR

Reply:
Yes but we won't allow it since we wants to stick to standards, the commentary is fine for me, it doesnt change anything and just explain some things

Reply:
You're right we can keep the `@private` even if we don't use JsDoc :)